components/Sponsors.jsx

```diff
@@ -0,0 +1,69 @@
+import Image from "next/image";
+import Link from "next/link";
+import Heading from "./Heading";
+import { GOLD_SPONSORS_ONE, GOLD_SPONSORS_TWO, SILVER_SPONSORS, BRONZE_SPONSORS } from "../data/sponsors";
+
+const SPONSORS_CONTAINER = "bg-white flex flex-col items-center py-20";
+const HEADING_CONTAINER = "flex flex-row justify-center mb-10";
+const SPONSORS_LEVEL_CONTAINER = "flex flex-row justify-center items-center my-5";
+const CONTENT_CONTAINER = "my-10 font-medium w-1/2 text-center ";
+const UNDERLINE_CONTAINER = "mb-10";
+const UNDERLINE_PROPERTIES = {
+  width: 177,
+  height: 10,
+};
+
+const Sponsors = () => {
+  const SponsorImage = ({ file, multiplier, height, website }) => (
+    <Link href={`https://www.${website}`}>
+      <a className="mx-5">
+        <Image src={`/sponsor_logos/${file}`} alt={file} width={height * multiplier} height={height} />
+      </a>
+    </Link>
+  );
+
+  return (
+    <div className={SPONSORS_CONTAINER}>
+      <div className={HEADING_CONTAINER}>
+        <Heading>Sponsors</Heading>

```

Comment:
On mobile, the sponsors title is slightly shifted left, could we align it a bit more right/center?

Reply:
Will do

Reply:
Check it now

Reply:
Looks good bro